app/models/condition.rb

```diff
@@ -2,6 +2,9 @@
 
 class Condition < ApplicationRecord
   belongs_to :planning_application
```

Comment:
Should we now also remove `belongs_to :planning_application`?

Or can a condition belong to both an application and a condition set? (It looks like planning application no longer `has_many` conditions.)

Reply:
You're right, I remember deleting that line! Maybe mixed it up on rebase 😄